CityPuzzle/Classes/Sql.cs

```diff
@@ -12,7 +12,7 @@ namespace CityPuzzle.Classes
 {
     class Sql
     {
- m        public static string ConnStr = "Server=tcp:citypuzzle.database.windows.net,1433;Initial Catalog=CityPuzzle;Persist Security Info=False;User ID=citypuzzle;Password=User123*;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;";
+        public static string ConnStr = "Server=tcp:citypuzzle.database.windows.net,1433;Initial Catalog=CityPuzzle;Persist Security Info=False;User ID=citypuzzle;Password=User123*;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;";
```

Comment:
It's nothing serious, but I think there should be spaces after semicolons and around "=" to maintain a consistent code style.

Reply:
Fixed and added line breaks for better readability

Reply:
Way more readable now.